<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountListDialog</class>
 <widget class="QDialog" name="AccountListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AccountListDialog{border:1px solid rgb(68,217,199);background-color:white;}
QLineEdit{border:1px solid rgb(214,214,214);border-radius:4px;padding: 0px 8px 0px 8px;font: 12x &quot;微软雅黑&quot;;}
QLabel{font: 12px &quot;微软雅黑&quot;;}</string>
  </property>
  <widget class="QTableWidget" name="accountListTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>780</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{border: 1px solid rgb(241,241,248);gridline-color: rgb(241,241,248)}
QHeaderView{background-color:white;min-height:30;}
QHeaderView::section{background-color: white;border: 1px solid rgb(241,241,248);
font: 12px &quot;微软雅黑&quot;;}</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>账户名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>地址</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>余额</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>当前账户</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="addAccountBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>81</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>新建账户</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>12</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{background-image:url(:pic2/close5.png);background-repeat: repeat-xy;background-position: center;background-attachment: fixed;background-clip: padding;border-style: flat;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>账户管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>113</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ff0000;&quot;&gt;沙盒模式下不能新建账户、导入账户&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>81</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QToolButton" name="importBtn">
   <property name="geometry">
    <rect>
     <x>754</x>
     <y>81</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>导入账户</string>
   </property>
  </widget>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>560</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>754</x>
     <y>560</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
